gum/backend-arm64/gumstalker-arm64.c

```diff
@@ -466,6 +466,31 @@ gum_stalker_set_trust_threshold (GumStalker * self,
   self->trust_threshold = trust_threshold;
 }
 
+void
+gum_stalker_flush (GumStalker * self)
+{
+  GSList * sinks, * cur;
+
+  GUM_STALKER_LOCK (self);
+
+  sinks = NULL;
+  for (cur = self->contexts; cur != NULL; cur = cur->next)
+  {
+    GumExecCtx * ctx = cur->data;
+	sinks = g_slist_prepend (sinks, g_object_ref (ctx->sink));
+  }
+
+  GUM_STALKER_UNLOCK (self);
+
+  for (cur = sinks; cur != NULL; cur = cur->next)
+  {
+    GumEventSink * sink = cur->data;
+	gum_event_sink_flush (sink);
```

Comment:
Should only use spaces for indentation:

```suggestion
    gum_event_sink_flush (sink);
```